README.rst

```diff
@@ -40,7 +40,7 @@ to secured endpoints should have an HTTP header called ``Authorization`` it must
 have as the value ``Bearer`` and the token returned by the login POST.
 
 New users may request an account via the web page hosted at ``/``. This form
-asks for the username and password they wish to use. New accounts are marked
+asks for the username and password they wish to use, as well as some other basic information. New accounts are marked
```

Comment:
Keep lines in rst and md files wrapped at 80 chars when possible. This reduces the clutter in git diffs